src/gwt/src/org/rstudio/studio/client/workbench/views/source/DocumentOutlineWidget.java

```diff
@@ -51,6 +50,7 @@
 import com.google.inject.Inject;
 
 public class DocumentOutlineWidget extends Composite
+                  implements EditorThemeStyleChangedEvent.Handler
```

Comment:
Do chunks need to listen for this event? (E.g. if you change the editor theme while you have a bunch of chunk outputs sitting in the IDE)